docs/guides/install-qiskit.mdx

```diff
@@ -13,11 +13,11 @@ Whether you will work locally or in a cloud environment, the first step for all
 <Admonition type="caution" title="Upgrade from Qiskit 0.x to Qiskit 1.0 and beyond">
   (If you are installing Qiskit for the first time, skip ahead to the [Install and set up](#local) section. This notice is relevant only to users who have installed Qiskit previously.)
 
-  **For those upgrading from version 0.x to 1.0 or later**: note that because Qiskit 1.0 uses a new packaging structure, you **cannot** use `pip install -U qiskit` to upgrade from any Qiskit 0.x version to 1.0. 
-  
-  See the [Qiskit 1.0 migration guide](/api/migration-guides/qiskit-1.0) for details and instructions. 
-  
-  Future updates starting with Qiskit 1.0 will allow for in-place upgrades.  
+  **For those upgrading from version 0.x to 1.0 or later**: note that because Qiskit 1.0 uses a new packaging structure, you **cannot** use `pip install -U qiskit` to upgrade from any Qiskit 0.x version to 1.0.
```

Comment:
```suggestion
  **For those upgrading from version 0.x to 1.0 or later**: note that because Qiskit v1.0 uses a new packaging structure, you **cannot** use `pip install -U qiskit` to upgrade from any Qiskit 0.x version to 1.0.
```